### PR TITLE
httpd.portal configuration improvements

### PR DESCRIPTION
**Pull request by dwlfrth**
# Description
- Cleaning up a bit httpd.portal config
- New configuration parameter for 'portal' interface type
- New configuration parameter for httpd.portal 'modstatus' port
# Impacts
- Webauth flow since we now need to add the 'portal' type to management interface if needed...
- httpd.portal since Apache configuration is modified
# Delete branch after merge

YES
# NEWS file entries
## New Features
- Introduced new 'portal' interface type to spawn a captive-portal instance on selected interface
- Introduced new 'ports.httpd_portal_modstatus' configuration parameter
## Enhancements
- Specific vhost for httpd.portal diagnostics

### CODE REVIEW

**Comment by dwlfrth** [comment]
Ready for review

**Comment by dwlfrth** [comment]
Please give opinion on possible "allow from" of the mod_status vhost...

**Comment by dwlfrth** [comment]
Review me please

**Comment by extrafu** [comment]
@fdurand Review and merge.

File: conf/documentation.conf [hunk 1]
```diff
@@ -655,6 +655,12 @@ description=<<EOT
 httpd listening port for PacketFence Status webservice
 EOT
 
+[ports.httpd_portal_modstatus]
+type=test
```
Comment (fdurand) [inline conf/documentation.conf]: text

**Pull request merged**